BlockchainTests/GeneralStateTests/stCallCreateCallCodeTest/Call1024OOG_d0g0v0.json

```diff
@@ -2,10 +2,10 @@
     "Call1024OOG_d0g0v0_Byzantium" : {
         "_info" : {
             "comment" : "",
-            "filledwith" : "cpp-1.3.0+commit.619a6438.Linux.g++",
-            "lllcversion" : "Version: 0.4.20-develop.2018.1.8+commit.2548228b.Linux.g++",
+            "filledwith" : "testeth 1.5.0.dev2-48+commit.413e4077.dirty",
+            "lllcversion" : "Version: 0.4.25-develop.2018.8.9+commit.63d071d6.mod.Linux.g++",
```

Comment:
Can you please not use an unstable nightly version of solidity/lll? :)

Reply:
I use custom version because it had wrong opcode for create2 I had to fix it manually. Need to check what current release version is.

Reply:
@axic, sorry, I cant use solidity develop. it has incorrect create2 being translated into 0xfb
the create2 should be translated into 0xf5 https://eips.ethereum.org/EIPS/eip-1014

Reply:
Fixed in https://github.com/ethereum/solidity/pull/5013.

However using a random nightly version can cause a variety of problems. I'd to either hold off refilling or take the release 0.4.25 and changing that single line in `libevmasm/Instruction.h`.

Reply:
smth changed. this lll translated into smth that cpp now dont understand. 
```{ (seq (CREATE2 0 0 (lll (seq (mstore 0 0x112233) (revert 0 32)) 0) 0) (SSTORE 0 (RETURNDATASIZE)) )}```

it was working before. Now I get invalid instruction